src/index.html

```diff
@@ -11,7 +11,145 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post"
+    >
+      <fieldset class="field">
+        <legend>Personal information</legend>
+        <label class="form">
+          Surname:
+          <input
+            type="text"
+            name="surname"
+            autocomplete="off"
+          >
+        </label>
+        <label class="form">
+          Name:
+          <input
+            type="text"
+            name="name"
+            autocomplete="off"
+          >
+        </label>
+        <label class="form">
+          How old are you?
+          <input
+            type="number"
+            name="age"
+            value="12"
+            min="1"
+            max="100"
+          >
+        </label>
+        <label class="form">
+          Full date of birth:
+          <input type="date" name="date">
+        </label>
+        <div class="form">
+          <label for="termsOfAgreement">
+            I accept the terms of the agreement
+          </label>
+          <input
+            type="checkbox"
+            name="terms"
+            id="termsOfAgreement"
+            required
+          >
+        </div>
+      </fieldset>
+      <fieldset class="field">
+        <legend>Registration</legend>
+        <label class="form">
+          E-mail:
+          <input
+            type="email"
+            name="email"
+            placeholder="email@example.com"
+          >
+        </label>
+        <label class="form">
+          Password
+          <input
+            type="password"
+            name="password"
+            minlength="5"
+            maxlength="10"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="field">
+        <legend>An interesting fact about you!</legend>
+        <label class="form">
+          Do you love cats?
+          <label>
+            <input
+              type="radio"
+              name="catLover"
+              value="yes"
+            >
+            Yes
+          </label>
+          <label>
+            <input
+                type="radio"
+                name="catLover"
+                value="no"
+            >
```

Comment:
Check the  code style here 
```suggestion
            <input
              type="radio"
              name="catLover"
              value="no"
            >
```